source/appModules/securecrt.py

```diff
@@ -16,7 +15,10 @@
 class AppModule(appModuleHandler.AppModule):
 
 	def chooseNVDAObjectOverlayClasses(self, obj, clsList):
-		if re.match("AfxFrameOrView\d{2,}u", obj.windowClassName) and obj.IAccessibleRole == oleacc.ROLE_SYSTEM_CLIENT:
+		#initialize a compiles regexp object for all window names following the standard SecureCrt Window naming comvention for all versions
+		MATCHER = re.compile ("AfxFrameOrView\d{2,}u")

```

Comment:
I moved the instantiation outside of the class. Sorry about that, didn't think about it being compiled over and over again. Makes sense and is now fixed.